scripts/convert_to_csv.py

```diff
@@ -0,0 +1,40 @@
+import pyreadr
+import glob, os
+
+
+errors = []
+
+for fn in glob.glob("../data/rda_files/*.rda"):
+    try:
+        data = pyreadr.read_r(fn, timezone="PST")
+    except Exception as e:
+        errors.append(os.path.basename(fn))
+        continue
+    if not data:
+        errors.append(os.path.basename(fn))
+        continue
+    for df in data.values():
+        if os.path.basename(fn) == "ncdc_normals.rda":

```

Comment:
One way to handle this slightly more generally would be to define a set of files at the top that need zipping:
```
NEED_ZIP = {"ncdc_normals.rda"}
```
Then the code here can change to 
```
if os.path.basename(fn) in NEED_ZIP:
```
That way if its ever decided to have more files be zipped, it is a confguration change at the top, rather than changes to the code.

Reply:
OK. Thank you. Will implement now...